Kitodo/src/main/java/org/kitodo/services/data/base/SearchService.java

```diff
@@ -233,6 +233,11 @@ public void save(T baseIndexedBean) throws DataException {
             saveToDatabase(baseIndexedBean);
             saveToIndex(baseIndexedBean);
             manageDependenciesForIndex(baseIndexedBean);
+            try {
```

Comment:
This and the similar changes in `remove` method looks like a workaround / hack for a problem. You should add at least a `TODO` for this as it could not fixed yet.

Reply:
It is not a hack or workaround. It is necessary to "give" time to ES to index all changes.